src/subscription_manager/scripts/rhsmcertd_worker.py

```diff
@@ -202,12 +202,12 @@ def main():
 
     parser = ArgumentParser(usage=USAGE)
     parser.add_argument("--autoheal", dest="autoheal", action="store_true",
-            default=False, help="perform an autoheal check")
+                        default=False, help="perform an autoheal check")
     parser.add_argument("--force", dest="force", action="store_true",
-            default=False, help=SUPPRESS)
+                        default=False, help=SUPPRESS)
     parser.add_argument(
-            "--auto-register", dest="auto_register", action="store_true",
-            default=False, help="perform auto-registration"
+        "--auto-register", dest="auto_register", action="store_true",
+        default=False, help="perform auto-registration"
```

Comment:
It is strange that there will be two types of indentation for same thing. Can you please fix it?

Reply:
PEP8 considers different styles for continuation lines depending on whether the first argument to a function call is in the same line or not: https://www.python.org/dev/peps/pep-0008/#indentation

I'm not changing that kind of continuation style for now, as there are tons of this occurrences to fix, in case.